### DISCUSSION TITLE
这个库是不是没人维护了，提了好几个issue都没人回复

### DISCUSSION THREAD

**Discussion by zym163** [vueComponent/ant-design-vue#7979] [General]
提了好几个issue都没人回复

**Comment by Ari1c** [comment]
感觉是的，开源是这样的

**Comment by tuskermanshu** [comment]
感觉是的 最近在调研 原本想用这个库 但是看到最新的一次更新还是2024年 就有点想放弃这个库了

**Reply by NoeyNoi** [reply]
> 感觉是的 最近在调研 原本想用这个库 但是看到最新的一次更新还是2024年 就有点想放弃这个库了

有bug，调研的时候没关注到实际已停更的状态，骑虎难下，有点不负责任

**Reply by wsxe9988** [reply]
一直没有停更，https://github.com/vueComponent/ant-design-vue/issues/8287
https://github.com/vueComponent/ant-design-vue/tree/feat/vapor
https://github.com/vueComponent/ant-design-vue/tree/feat-4.3

**Comment by yuantongkang** [comment]
什么时候开始维护 ant design vue

**Comment by wsxe9988** [comment]
https://github.com/vueComponent/ant-design-vue/issues/8287

**Comment by NoeyNoi** [comment]
https://github.com/surely-vue/surely-table/commits/main/ 看到这个商业库2025 年 7 月 15 日还有一次提交，应该是只关注商业化了，非常感谢作者的开源行为，但是停更没有通知确实有点不负责任！！！！！

**Reply by wsxe9988** [reply]
> https://github.com/surely-vue/surely-table/commits/main/ 看到这个商业库2025 年 7 月 15 日还有一次提交，应该是只关注商业化了，非常感谢作者的开源行为，但是停更没有通知确实有点不负责任！！！！！

一直没有停更，https://github.com/vueComponent/ant-design-vue/issues/8287  
https://github.com/vueComponent/ant-design-vue/tree/feat/vapor
https://github.com/vueComponent/ant-design-vue/tree/feat-4.3
